src/options/output.rs

```diff
@@ -0,0 +1,203 @@
+use std::str::FromStr;
+
+use anyhow::Result;
+use calm_io::{stderrln, stdoutln};
+use camino::Utf8PathBuf;
+use clap::{error::ErrorKind as ClapErrorKind, CommandFactory, Parser, ValueEnum};
+use rover_client::shared::SdlType;
+use rover_std::{Fs, Style};
+use serde::Serialize;
+
+use crate::{
+    cli::{FormatType, Rover},
+    command::output::JsonOutput,
+    RoverError, RoverOutput, RoverResult,
+};
+
+#[derive(Debug, Parser)]
+pub struct Output {
+    /// The file path to write the command output to.
+    #[clap(long)]
+    output: OutputType,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize)]
+pub enum FinalOutputType {
+    File(Utf8PathBuf),
+    Stdout,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize)]
+pub enum OutputType {
+    LegacyOutputType(FormatType),
+    File(Utf8PathBuf),
+}
+
+impl FromStr for OutputType {
+    type Err = anyhow::Error;
+
+    // TODO: HANDLE THE TRANSFORMATION OF THE JSON LEGACY FORMAT TYPE
```

Comment:
did we handle this TODO? if so, we can remove this comment

Reply:
Ah crumbs we did! I'll remove 👍